### PR TITLE
Renamed Renga->Renku

### PR DESCRIPTION
**Pull request by leafty**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ sudo: true
 env:
   global:
   - DOCKER_LABEL=$TRAVIS_BRANCH
-  - DOCKER_REPOSITORY=rengahub/
+  - DOCKER_REPOSITORY=renkuhub/
```
Comment (leafty) [inline .travis.yml]: ->renku

File: Makefile [hunk 2]
```diff
@@ -15,9 +15,9 @@
 # limitations under the License.
 
 # Define docker image tag
-DOCKER_REPOSITORY?=rengahub/
+DOCKER_REPOSITORY?=renkuhub/
```
Comment (leafty) [inline Makefile]: ->renku

File: README.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
-# renga-storage
-Renga Storage Service
+# renku-storage
+Renku Storage Service
 
-**Renga is currently undergoing a major restructuring effort. For a preview, you can look
+**Renku is currently undergoing a major restructuring effort. For a preview, you can look
 at the development branch, but keep in mind it is highly volatile.**
```
Comment (leafty) [inline README.md]: what to do about this notice?

File: README.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
 
-Documentation: https://renga.readthedocs.io/en/latest/developer/services/storage_service.html
+Documentation: https://renku.readthedocs.io/en/latest/developer/services/storage_service.html
```
Comment (leafty) [inline README.md]: keep old?

File: README.md (hunk 3 above)
Comment (rokroskar) [reply README.md]: I would remove this notice

File: README.md (hunk 4 above)
Comment (rokroskar) [reply README.md]: we can for now, but a new description will be needed in the docs

File: README.md [hunk 5]
```diff
@@ -59,5 +59,5 @@ by setting the environment variable `STORAGE_BACKEND_LOCAL_ENABLED` to `true`.
 | GITLAB_URL |  | the root url of gitlab |
 | GITLAB_USER |  | the gitlab user |
 | GITLAB_PASS |  | the gitlab user's private token |
-| RENGA_ENDPOINT | http://localhost | the url to reach renga from the users perspective |
-| RENGA_STORAGE_DEFAULT_LFS_BACKEND | local | the default backend to create buckets if none is specified for a git repository |
+| Renku_ENDPOINT | http://localhost | the url to reach renku from the users perspective |
```
Comment (ebiiii) [inline README.md]: Don't we keep all caps for the env variable names here ?

File: conf/application.conf [hunk 6]
```diff
@@ -56,11 +56,11 @@ storage.backend.localgit.enabled = ${?STORAGE_BACKEND_LOCALGIT_ENABLED}
 storage.backend.localgit.root = "/data/repo"
 storage.backend.localgit.root = ${?LOCAL_STORAGE_GIT_ROOT}
 
-renga_host = "http://localhost"
-renga_host = ${?RENGA_ENDPOINT}
+renku_host = "http://localhost"
+renku_host = ${?Renku_ENDPOINT}
```
Comment (ebiiii) [inline conf/application.conf]: if changed above, should also update here

File: README.md [hunk 7]
```diff
@@ -19,10 +16,10 @@ Alternatively, you can use the Makefile to get git branch and commit based tags:
 ```bash
 make
 > Successfully built cb53b5cd36eb
-> Successfully tagged rengahub/renga-storage:08a076cf5c49
-> docker tag rengahub/renga-storage:08a076cf5c49 rengahub/renga-storage:development
+> Successfully tagged renkuhub/renku-storage:08a076cf5c49
+> docker tag renkuhub/renku-storage:08a076cf5c49 renkuhub/renku-storage:development
 ```
-Two tags were defined: `rengahub/renga-storage:08a076cf5c49` and `rengahub/renga-storage:development` where `08a076cf5c49` is the commit sha1 (truncated) and `development` is the git branch name.
+Two tags were defined: `renkuhub/renku-storage:08a076cf5c49` and `renkuhub/renku-storage:development` where `08a076cf5c49` is the commit sha1 (truncated) and `development` is the git branch name.
```
Comment (rokroskar) [inline README.md]: make sure the repository name is renku not renkuhub everywhere

File: README.md [hunk 8]
```diff
@@ -1,8 +1,5 @@
-# renga-storage
-Renga Storage Service
-
-**Renga is currently undergoing a major restructuring effort. For a preview, you can look
-at the development branch, but keep in mind it is highly volatile.**
+# renku-storage
+Renku Storage Service
 
 Documentation: https://renga.readthedocs.io/en/latest/developer/services/storage_service.html
```
Comment (rokroskar) [inline README.md]: renga --> renku

File: README.md (hunk 4 above)
Comment (rokroskar) [reply README.md]: renku docs build now so this change is good to go

**Pull request merged**